user-guide/Advanced_Functionality/Failover/Viewing_Failover_information/Viewing_the_current_Failover_DMA_status.md

```diff
@@ -65,5 +65,13 @@ This will open the *Status* dialog box, which displays a number of status parame
 
     “Failing” means that the database server is probably not running or that there is a configuration issue.
 
+- **Packet capture libraries**: "Pcap not installed" or "Could not be detected"
+
+  - "Pcap not installed" means that neither Npcap nor WinPcap is installed on the DMA. In this case, we do not recommend using this Agent in a Failover setup with virtual IP addresses.
+
+  - "Could not be detected" means there is a problem detecting the presence of the Npcap and WinPcap libraries.
+
+  - If no message is displayed, this indicates that either Npcap (recommended) or WinPcap (deprecated) is installed on the DMA. In this case, the Agent is suitable for a Failover setup with virtual IP addresses.
```

Comment:
@HansSolos, to my knowledge, the status info here applies to both DMAs, so should we adjust this like this? 
```suggestion
  - "Pcap not installed" means that neither Npcap nor WinPcap is installed on the DMAs. In this case, we do not recommend using these Agents in a Failover setup with virtual IP addresses.

  - "Could not be detected" means there is a problem detecting the presence of the Npcap and WinPcap libraries.

  - If no message is displayed, this indicates that either Npcap (recommended) or WinPcap (deprecated) is installed on the DMAs. In this case, the Agents are suitable for a Failover setup with virtual IP addresses.
```